values.yaml

```diff
@@ -1,3 +1,4 @@
+enable_intermediate_model_removal: False
```

Comment:
You may also need to add this to local-values.yaml.

Reply:
Will do, thank you!

Reply:
Fixed: https://github.com/SubstraFoundation/substra-tests/pull/44/commits/23c97c74d6ad2a365535a1b2f8619f8fa8fdca99